azure-pipelines.yml

```diff
@@ -589,7 +589,6 @@ jobs:
         **\WebJobs.Script.Tests.Integration.csproj
   - task: DotNetCoreCLI@2
     displayName: "Release verification tests"
-    condition: eq(variables['isReleaseBranch'], 'True')
```

Comment:
my recommendation even though test is not perfect - we should some format that verifies checked in file with the code.
Also, leave this test in release branch only to ensure PR builds are not blocked. But add the file read test in dev branch

cc @safihamid